app/src/main/java/helium314/keyboard/keyboard/internal/keyboard_parser/LocaleKeyboardInfos.kt

```diff
@@ -341,7 +330,7 @@ const val POPUP_KEYS_NORMAL = "normal"
 private const val LOCALE_TEXTS_FOLDER = "locale_key_texts"
 
 // either tld is not simply lowercase ISO 3166-1 code, or there are multiple according to some list
-private val specialCountryTlds = listOf(
+private val specialCountryTlds = mapOf<String, String>(
```

Comment:
Another tiny thing: default map implementation in Kotlin is [`LinkedHashMap`](https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/src/kotlin/collections/Maps.kt#L54). If you don't need a specific order in the map, `hashMapOf` is a more efficient choice.
But that's just additional information, in this specific case I don't think the difference is relevant.